jenkins/freestyle/get-sitespeed-report.sh

```diff
@@ -35,84 +35,88 @@ set -e
 : ${CONNECTION:?"The CONNECTION environment variable must be set."}
 : ${SITESPEED_BROWSER:?"The SITESPEED_BROWSER environment variable must be set."}
 
-if [ $GET_URLS_FROM_FILE == "true" ] ; then
-	msg="The SITESPEED_URL_FILE environment variable must be set to the file containing the URLs to test."
-	: ${SITESPEED_URL_FILE:?$msg}
+# Test for the existence of TEST_URL_FILE
+if [ ! -z ${TEST_URL_FILE+x} ] ; then
```

Comment:
This pattern tests that there is an environment variable, and that it's not empty.